distribution/docs/src/main/resources/_contents/_broker/route-manager-contents.adoc

```diff
@@ -0,0 +1,21 @@
+==== Route Manager
+
+===== Prerequisites for Route Manager
+
+None
+
+===== Installing and Uninstalling
+
+The Route Manager is installed as a part of the Broker App.
+
+===== Configuring
+
+The route shutdown timeout can be configured.
+
+===== Using the Route Manager
+
+The Route Manager gives an administrator the ability to configure and deploy Camel routes, queues, and topics dynamically. The `sjms` component is available by default. If a need arises for a new route, an administrator can easily develop a new route and deploy it to satisfy the requirement, rather than spending the time to develop, compile, and test new code.
```

Comment:
This section should probably come sooner (line 2-ish) as an intro to this section as it defines the route manager.